schemas/chemotherapy.json

```diff
@@ -27,7 +27,8 @@
         "validationDependency": true,
         "primaryId": true,
         "foreignKey": "sample_registration.submitter_donor_id",
-        "displayName": "Submitter Donor ID"
+        "displayName": "Submitter Donor ID",
+        "notes": "Identifier cannot begin with prefix 'DO'."
```

Comment:
@hknahal we can do this with a reference as well in case we need to edit this message ever. 

It works the same way as the script references.

Reply:
@rosibaj How do I do the reference? I don't see an example of it in the dictionary

Reply:
Do you mean create a list of note references in the 'references' directory and then refer to it in the dictionary the same way we do with codelists?

Reply:
https://github.com/icgc-argo/argo-dictionary/pull/193  <-- this is how